common/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -309,6 +318,11 @@ export class ActiveKey implements LayoutKey {
   }
 }
 
+
+export class ActiveSubkey extends ActiveKeyBase implements LayoutSubKey {
```

Comment:
```suggestion
export class ActiveSubKey extends ActiveKeyBase implements LayoutSubKey {
```

Of course this will have ripple effects on this PR so you can't just apply this suggestion without fixing all the refs but I think it's important to be consistent on this

Reply:
Oh.  I somehow thought I'd _introduced_ that class in this PR.  Surprised I didn't note that on my usual re-review; I'll see about reversing it.